Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Io_image_plugin.cpp

```diff
@@ -222,7 +222,7 @@ class Io_image_plugin :
       connect(planeSwitch, SIGNAL(triggered()),
               this, SLOT(selectPlanes()));
       Scene* true_scene = dynamic_cast<Scene*>(scene);
-      connect(true_scene,SIGNAL(selectionChanged(int)),
+      connect(true_scene,SIGNAL(itemIndexSelected(int)),
```

Comment:
I hope that is documented correctly, because that is really confusing, and the names of the two functions, and the names of the two signals are two similar.